### ISSUE TITLE
Semantic Markup for the "Last modified" date for an article

### ISSUE THREAD

**Issue by saschaleib** [dokuwiki/dokuwiki#4340]
The "last modified" date that is added to the bottom of the the pages in the default template lacks the semantic markup to include a machine-readable representation of the date.

This section should be marked up using the <[time](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)> element.

In addition, the `datetime` attribute should be used to insert an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) formatted date/time string.

This will make it possible to process the modification date programmatically, even in situations where the chosen display format is unpredictable and/or complex.

Expected output for this field, e.g.:

```
<time datetime="2024-11-03T09:58:28+0200">Sunday, 3 November 2024 at 09:58:28 AM</time>
```

**Comment by fiwswe** [comment]
This is apparently fixed in "Librarian".

Can this be closed?


### Tested on
Release 2025-05-14b "Librarian"
PHP 8.5.8
OpenBSD 7.9
fpm-fcgi

Release 2026-07-14 "Mort"
sprintdoc dw
PHP 8.5.8 fpm-fcgi
OpenBSD 7.9

**Comment by saschaleib** [comment]
Indeed, it appears to be fixed on the DokuWiki site (which I assume always uses the latest DW version). Good job! :-)
